layouts/maternity-care-sharing-info2.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Combine all permutations of a layout into the same file.

See https://github.com/pebcakerror/layouts/blob/master/layouts/video-section.html or https://github.com/pebcakerror/layouts/blob/master/layouts/page-header.html